examples/amr_human_paired_sample_with_rpm.json

```diff
@@ -1,7 +1,7 @@
 {
-  "output_dir_s3": "s3://idseq-samples-prod/samples/12/5815/postprocess_test",
+  "output_dir_s3": "s3://idseq-samples-test/lvreynoso/amr",
```

Comment:
Do not use your user name here. Just move the files into a more generic folder if necessary.